include/mender-storage.h

```diff
@@ -81,7 +81,7 @@ mender_err_t mender_storage_get_deployment_data(char **deployment_data);
  * @param artifact_type artifact type of the update module we are saving the state of
  * @param state state of the update module
  */
-mender_err_t mender_storage_save_update_state(mender_update_state_t state, const char *artifact_type);
+mender_err_t mender_storage_save_update_state(const mender_update_state_t state, const char *artifact_type);
```

Comment:
This has no effect here in the function declaration.

Reply:
Do you mean that it is passed by copy? I'll remove it as this is not really what the PR was about but I don't understand what you mean :man_student:

Reply:
What he means is that the `const` keyword (in this case) has no effect in the function declaration. If you put it in the function definition, it will have an effect.

Reply:
Right. But it was both in the declaration and in the definition (both for Zephyr and POSIX). I did revert this, anyway.